control-plane/pkg/reconciler/consumergroup/consumergroup.go

```diff
@@ -341,12 +347,30 @@ func (r Reconciler) schedule(ctx context.Context, cg *kafkainternals.ConsumerGro
 	return nil
 }
 
+func (r *Reconciler) maybeRequeueOnPendingReplicas(cg *kafkainternals.ConsumerGroup) {
```

Comment:
`requeueOnNotMachtingReplicaCount`?  I think that reflects a bit better the comment in the func.